test/refactoring/ConvertAsync/convert_bool.swift

```diff
@@ -0,0 +1,271 @@
+// REQUIRES: objc_interop
+// REQUIRES: concurrency
+
+// RUN: %empty-directory(%t)
+
+import Foundation
+import ConvertBoolObjC
+
+func boolWithErr(completion: (Bool, Error?) -> Void) {}
+func multipleBoolWithErr(completion: (String?, Bool, Bool, Error?) -> Void) {}
+func optionalBoolWithErr(completion: (String?, Bool?, Bool, Error?) -> Void) {}
+
+// All 4 of the below should generate the same refactoring.
+
+// RUN: %refactor -convert-call-to-async-alternative -dump-text -source-filename %s -pos=%(line+1):1 -I %S/Inputs -sdk %clang-importer-sdk-path  | %FileCheck -check-prefix=BOOL-WITH-ERR %s
```

Comment:
These don't need -I or the SDK path right?

Reply:
They're needed to resolve the imports in the file, though it seems `swift-refactor` will still return 0 in that case, still I think it it's probably best to include them so the errors don't clutter the stderr when debugging a failure, what do you think?

Reply:
Ah true, yep - keeping them sounds good.